### PR TITLE
Help javascript developpers find where to run "npm commands"

### PR DESCRIPTION
**Pull request by Hyzual**

### CODE REVIEW

File: languages/en/developer-guide/front-end/javascript.rst [hunk 1]
```diff
@@ -37,9 +37,51 @@ While you are working, in the appropriate "scripts" folder, run the following co
 		* ``npm run test`` will run the unit tests once. It is used by the Continuous integration to validate changes.
 		* ``npm run coverage`` will run the unit tests, generate HTML reports and open your default browser with those reports. You can use it to navigate in the code and check out how well unit tests cover the production code.
 
+Where can I run npm commands ?
+------------------------------
+
+Tuleap has a long history and as more functionalities are added, more javascript is added too. Nowadays, it can get hard to know where to run ``npm run watch`` in all those plugins.
```
[the remark this answers is not in the document]
Comment (Hyzual) [reply languages/en/developer-guide/front-end/javascript.rst]: I think this formulation is also okay. I'll change it though

File: languages/en/developer-guide/front-end/javascript.rst [hunk 2]
```diff
@@ -37,9 +37,51 @@ While you are working, in the appropriate "scripts" folder, run the following co
...
+The safest way to know is to find ``package.json`` files. Here's a bash command to let you do that:
+
+	.. code-block:: bash
+
+		$ find . -type f -name 'package.json' -print -o -path '*/node_modules' -prune
+
+This command returns all the folders containing a ``package.json`` file. You can run npm commands in them. Remember that not all npm scripts will be implemented, some of these folder may only have a "build" script.
+
+.. code-block:: text
```
[the remark this answers is not in the document]
Comment (gorkat) [reply languages/en/developer-guide/front-end/javascript.rst]: What is the purpose of having a list and a command that give you the list? The command will always give all the locations, but the list only if it is up to date.

[the remark this answers is not in the document]
Comment (Hyzual) [reply languages/en/developer-guide/front-end/javascript.rst]: I intentionally removed our Enalean :tm: internal plugins from the list. The command will list them for you though.
What do you want me to do ?

[the remark this answers is not in the document]
Comment (gorkat) [reply languages/en/developer-guide/front-end/javascript.rst]: Remove the list, if it is ok for everybody

**Comment by LeSuisse** [comment]
I'm going to merge that then drop the list to make everyone almost happy.

**Pull request merged**